src/library/returnBook/ReturnBookControl.java

```diff
@@ -0,0 +1,82 @@
+package library.returnBook;
+import library.entities.Book;
+import library.entities.Library;
+import library.entities.Loan;
+
+public class ReturnBookControl {
+
+    private ReturnBookUI Ui;
+    private enum ControlState {INITIALISED, READY, INSPECTING};
+    private ControlState state;
+    
+    private Library library;
+    private Loan currentLoan;
+    
+
+    public ReturnBookControl() {
+        this.library = Library.getInstance();
+        state = ControlState.INITIALISED;
+    }
+    
+    
+    public void sEt_uI(ReturnBookUI ui) {

```

Comment:
"sEt_uI" should have appropriate upper and lowercase formatting as well as having the underscore removed.

Reply:
Resolution: https://github.com/9Mcgusto6/ITC205-Team17/pull/5/commits/f42b05350a59912c8be10c841c9be3d554c85678